src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -44,6 +49,14 @@
  * @since 0.7
  */
 public final class ManifestsTest {
+    // @checkstyle JavadocVariableCheck (8 lines)
+    public static final String TEST_MANIFEST = "test.mf";
+    public static final String TEST_MANIFEST_2 = "another-test.mf";
+    public static final String TEST_ATTRIBUTE = "some test attribute";
+    public static final String TEST_ATTRIBUTE_2 = "another attribute";
+    public static final String FROM_FILE = "From-File";
+    public static final String VERSION = "Version";
```

Comment:
@nesterov-n please see that article, we should not have such constants: http://www.yegor256.com/2015/07/06/public-static-literals.html